built_value/lib/standard_json_plugin.dart

```diff
@@ -59,6 +78,44 @@ class StandardJsonPlugin implements SerializerPlugin {
     return result;
   }
 
+  Map _toMapSpecifyType(List list) {
+    var type = list[0];
+
+    if (list.length == 2) {
```

Comment:
can list.length == 1?

Reply:
Nope; the list contains at least a type name and a value. Added a comment.